### ISSUE TITLE
[Feature Request] Show Pomotroid window when timer ends

### ISSUE THREAD

**Issue by camayl** [Splode/pomotroid#221]
**Feature:**

- "show Pomotroid window when timer ends", and alternatively, 
- "hide Pomotroid timer when timer starts". 

**Reasoning:** 
This sounds like a minor and unimportant feature, but I noticed that when I am really focused on other tasks on my computer, I tend to forget about the timer running, even when the alarm sounds and the break starts. This causes me to skip breaks and as a consequence get burnt out quicker. 
Having the Pomotroid window appear when the work timer ends, would facilitate sticking to the work and break times immensely for me.

**Comment by Gr-Nobel** [comment]
Yes another app called Promataz has a functionality to use full screen breaks. 
![image](https://user-images.githubusercontent.com/95166364/200745830-924f29d5-2c67-4626-8836-db163a334810.png)
